utils/check_config_docstrings.py

```diff
@@ -73,7 +74,9 @@ def get_checkpoint_from_config_class(config_class):
 def check_config_docstrings_have_checkpoints():
     configs_without_checkpoint = []
 
-    for config_class in list(CONFIG_MAPPING.values()):
+    a = [CTRLConfig]
```

Comment:
Looks like this debugging statement made it into the final PR! Opened a PR to revert these changes: